src/action.ts

```diff
@@ -78,16 +77,21 @@ async function fetchOrLocateSonarResultsFile(resultType : SONAR_RESULT) {
   return fetchOrLocateResultsFile("sonar", results, fileName);
 }
 
-async function fetchOrLocateResultsFile(tool: Tool, results: any, fileName: string) {
+async function fetchOrLocateResultsFile(tool: string, results: any, fileName: string, stringifyResults: boolean = true) {
```

Comment:
Can you update the default case (Line 45), so only calls the core.getInput, since it will never do the logic about the writign the file